deploy/monai-deploy-express/configs/task-manager.json

```diff
@@ -28,12 +28,17 @@
       "topics": {
         "workflowRequest": "md.workflow.request",
         "exportComplete": "md.export.complete",
-        "exportRequestPrefix": "md.export.request"
+        "exportRequestPrefix": "md.export.request",
+        "taskCallback": "md.tasks.callback"
       },
       "dicomAgents": {
         "dicomWebAgentName": "monaidicomweb",
         "scuAgentName": "monaiscu"
       },
+      "argoCallback": {
```

Comment:
NIT: Argo is not supported in MD Express, correct? Leaving it here is fine, though might confuse users who actually look into this.

Reply:
I will leave it there to avoid breaking TM.

Reply:
Understood. Looks TM is a little fragile if true.